final/client/src/containers/cart-item.tsx

```diff
@@ -14,7 +15,9 @@ export const GET_LAUNCH = gql`
   ${LAUNCH_TILE_DATA}
 `;
 
-export default function CartItem({ launchId }) {
+interface CartItemProps extends LaunchDetailTypes.LaunchDetailsVariables {}
+
+const CartItem: React.FC<CartItemProps> = ({ launchId }) => {
```

Comment:
What's your thought process behind extending the `LaunchDetailTypes.LaunchDetailsVariables` interface here? If there were additional props we had to declare, I could see why you would. Since there aren't additional props, we should be able to type `CartItem` as `React.FC<LaunchDetailTypes.LaunchDetailsVariables>`.

Reply:
@peggyrayzis My reasoning was to keep the naming structure of props for a component consistent (like what was done for `LaunchTile`, `LaunchDetails`, etc).

My personal preference is that I like to ship a component _with_ the type or interface that describes its inputs, _composed_ by types from the models (or in this case, the query variables).

Your suggestion is functionally equivalent to what I have here so we could do either one.

Reply:
Awesome! On second thought, I like the separation of concerns. Let's go with it for all other components.